ImagePicker/ImagePickerControllerDelegate.swift

```diff
@@ -17,6 +17,12 @@ public protocol ImagePickerControllerDelegate: class {
 
     /// Called when user takes new photo.
     func imagePicker(controller: ImagePickerController, didTake image: UIImage)
+    
+    /// Called when user takes new live photo.
+    func imagePicker(controller: ImagePickerController, didTakeLive livePhoto: UIImage, videoUrl: URL)
```

Comment:
the api could be uniform. Either use 
func imagePicker(controller: ImagePickerController, didTake livePhoto: UIImage, videoUrl: URL)

or func imagePicker(controller: ImagePickerController, didTakeLivePhoto livePhoto: UIImage, videoUrl: URL)